layouts/filter.tsx

```diff
@@ -8,14 +8,13 @@ import FilterHeader from "components/molecules/FilterHeader/filter-header";
 
 import useNav from "lib/hooks/useNav";
 
-const FilterLayout = ({children}: {children: React.ReactNode}) => {
-
+const FilterLayout = ({ children }: { children: React.ReactNode }) => {
   const { toolList, selectedTool, filterName, selectedFilter, userOrg } = useNav();
 
   return (
     <>
       <TopNav />
-      <div className="page-container flex min-h-[calc(100vh-(54px+95px))] flex-col items-center">
+      <div className="page-container flex min-h-[calc(100vh-(54px+95px))] flex-col items-center overflow-x-hidden">
```

Comment:
Thanks for opening a PR for this, however hiding the overflow is not the correct way to solve this problem; we should figure out what the underlying element that breaks the 100% width range and assess whether this is a platform bug.

For reference, can you let us know your operating system and browser so we can replicate a testing environment? 🍕

Reply:
sure, i'm on windows 11 using a chrome browser, overflow appears around 1300px. following your advice I took a closer look and noticed that changing the tooltip nav.tsx parent elements width property from 100vw to 100% also fixes the overflow without having to hide it.

Reply:
hi @0-vortex you can review the changes I made?